.github/workflows/quick_start.yml

```diff
@@ -0,0 +1,11 @@
+name: quick-start
+
+on: [push]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run quick_start.sh
+        run: ./bin/quick_start.sh
```

Comment:
Do we need checkout? We're downloading the docker-compose file?
and the quickstart script cannot be done with run-statements?

Reply:
the github action is changed to use only run statements